apps/web-mzima-client/src/app/shared/components/search-form/search-form.component.html

```diff
@@ -25,6 +25,7 @@
       class="search-form__form-control"
       [ngModelOptions]="{ standalone: true }"
       [placeholder]="'global_filter.search' | translate"
+      [data-qa]="'search-form__search-posts'"
```

Comment:
Could you rewrite the [data-qa] unique attribute name with dashes "-" and not underscores "_".